src/opm/parser/eclipse/Deck/Deck.cpp

```diff
@@ -30,143 +30,76 @@ namespace fs = std::filesystem;
 
 namespace Opm {
 
-    bool DeckViewInternal::hasKeyword( const std::string& keyword ) const {
-        return this->keywordMap.find( keyword ) != this->keywordMap.end();
+    const std::vector< const DeckKeyword* > Deck::getKeywordList( const std::string& keyword ) const {
```

Comment:
Note that returning a `const vector<>` here inhibits move semantics.  Is that intentional?

Reply:
Nope